pbsmetrics/metrics.go

```diff
@@ -19,7 +23,11 @@ type Labels struct {
 // AdapterLabels defines the labels that can be attached to the adapter metrics.
 type AdapterLabels struct {
 	Source        DemandSource
-	RType         RequestType
+	RType         RequestType //change??
```

Comment:
Request type is something entirely different from the Imp type, so no need to change